modules/datetime.js

```diff
@@ -0,0 +1,10 @@
+// eslint-disable-next-line import/no-unresolved
+import { DateTime } from '../../node_modules/luxon/src/luxon.js';
```

Comment:
- [ ] This disabling is also unnecessary. Kindly remove it. You simply need to fix your import path.

  ```
  import { DateTime } from '../node_modules/luxon/src/luxon.js';
  ```